raiden/ui/cli.py

```diff
@@ -597,13 +598,22 @@ def run_app(
     privatekey_hex = hexlify(privatekey_bin)
     config['privatekey_hex'] = privatekey_hex
 
-    rpc_host, rpc_port = eth_endpoint_to_hostport(eth_rpc_endpoint)
+    web3 = Web3(HTTPProvider(eth_rpc_endpoint))
+
+    try:
+        node_version = web3.version.node  # pylint: disable=no-member
+    except ConnectTimeout:
+        raise EthNodeCommunicationError('couldnt reach the ethereum node')
```

Comment:
Not sure if we display this anywhere, but I would write *Couldn't connect to the ethereum node*